server/app/views/admin/programs/ProgramPredicateConfigureView.java

```diff
@@ -515,6 +515,9 @@ private DivTag createScalarDropdown(
     } else {
       try {
         scalars = Scalar.getScalars(questionDefinition.getQuestionType());
+        if (questionDefinition.getQuestionType().equals(QuestionType.NAME)) {
```

Comment:
**suggestion:** This can be after the `if (questionDefinition.isAddress()) {` before the `else`. With the current implementation we're making a call to get the scalars and not using them. If we need `maybeSelectedScalar` that could be defined within the if as well, but I'm not sure we need that?

Reply:
I think this should stay inside of `else` to ensure that `maybeSelectedScalar` is initialized. Unless you see any benefit taking it out of `else`?

Reply:
Ah got it makes sense. It was just a little weird to me to have scalars predefined then rewritten so we could consider putting scalars (the line above the if) into an else within this else, but fine either way. Makes sense that we need to define maybeSelectedScalar